src/main/java/com/ranorex/jenkinsranorexplugin/RanorexRunnerBuilder.java

```diff
@@ -187,13 +290,24 @@ public String getCmdLineArgs() {
      *                              so, so that users have better understanding on why it failed.
      */
     @Override
-    public boolean perform(AbstractBuild<?, ?> build, Launcher launcher, BuildListener listener) throws IOException, InterruptedException {
-        jArguments = new ArgumentListBuilder("cmd.exe", "/C");
-        WorkSpace = FileUtil.getRanorexWorkingDirectory(build.getWorkspace(), rxTestSuiteFilePath).getRemote();
+    public void perform(@Nonnull Run<?, ?> run, @Nonnull FilePath workspace, @Nonnull Launcher launcher,
+            @Nonnull TaskListener listener) throws InterruptedException, IOException {
+    	EnvVars env;
```

Comment:
Indentation messed up in this line and 10 lines below

Reply:
Fixed.